### ISSUE TITLE
Library for performing queries over TUX database

### ISSUE THREAD

**Issue by FAMILIAR-project** [TuxML/tuxml-web#3]
We have to split the work in two: 
 * part of the team on the interface (everything HTML/CSS/JS) and integration of the queries
 * another part of the team on the development of a set of Python functions, parameterized, that implement the queries
these functions can be called either via the server (and communication with the JS) or offline, to pre-calculate results.
functions 

I'm thinking of: count_kernels(version), count_errors(version), etc.
Basically a .py file with a lot of reusable Python functions: a library to query the TUXML database.

Who wants to be a developer of this file?
